spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_4_1_spec.rb

```diff
@@ -206,8 +206,8 @@ def assert_specific_host
       :cpu_type             => "Intel(R) Xeon(R) CPU           E5504  @ 2.00GHz",
       :manufacturer         => "",
       :model                => "",
-      :number_of_nics       => nil,
-      :memory_mb            => 56333,
+      :number_of_nics       => 2,
+      :memory_mb            => 7806,
```

Comment:
Oh I get it now. On this branch both refreshers (3_1 and 4_1) do the same thing but the vcr recorded for 4_1 is not yet represented properly in the refresher_4_1_spec. The correctness of the nics change is checked in refresher_3_1_spec so I am removing this changes from here (the line that runs this assertion is commented out anyway).